index.html

```diff
@@ -29,7 +36,15 @@
       <!---------------- Mobile Menu ------------>
       <header class="pf-header">
         <a class="pf-logo" href=""><img class="pf-logo" src="./img/F_logo.png" alt="Fahim logo" /></a>
-        <nav class="material-symbols-outlined">menu</nav>
+        <a href="#" class="" id="toggleMenuButton"><i class="fa fa-bars" id="m-icon" aria-hidden="true"></i></a>
+        <nav hidden class="menu-show" id="toggleMenuItems">
+          <a href="#"><i class="fa fa-times" id="x-icon" aria-hidden="true"></i></a>
+          <ul class="coll-nav-items">
+            <li><a href="#portfolioSection">Portfolio</a></li>
+            <li><a href="#aboutSection">About</a></li>
+            <li><a href="#formSection">Contact</a></li>
+          </ul>
+        </nav>
```

Comment:
- I see that you have two `nav` tags in your HTML page, this is not good practice, to help improve the structure of your page I suggest that you have one `nav` tag and style it using media-queries to fit into mobile or desktop version. Please fix this 🙏 

- Also good job done on your mobile menu, but there are some differences between your project and what is in the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
-- There should be a backdrop-filter that blurs the main page when the mobile menu is open. Please fix this 🙏 
<img width="270" alt="3grgerg" src="https://user-images.githubusercontent.com/73607512/217212407-736a3144-cf80-4454-bf7d-ed13e9cb1a4b.PNG">

Figma design:
<img width="144" alt="nedtaethe" src="https://user-images.githubusercontent.com/73607512/217212963-116519d7-cf85-41b3-9a66-7640b1189055.PNG">

Reply:
resolved! ✔️